packages/core/scripts/setup/android.ts

```diff
@@ -112,31 +112,20 @@ export const createEmbraceJSON = {
     'https://embrace.io/docs/react-native/integration/add-embrace-sdk/?platform=android#manually',
 };
 
-const embraceInit =
-  'Embrace.getInstance().start(this, false, Embrace.AppFramework.REACT_NATIVE);';
-const embraceImport = 'import io.embrace.android.embracesdk.Embrace;';
+const tryToPatchMainApplication = () => {
+  // In the future there will be more apps with kotlin than java so I start looking up by kotlin
+  const response = patch('kotlin');
+  if (!response) {
+    return patch('java');
+  }
+  return response;
+};
+
 export const patchMainApplication = {
-  name: 'patch MainApplication.java file',
+  name: 'patch MainApplication.kt file',
```

Comment:
could be either .java or .kt right? Maybe we just say "patch MainApplication file" with no extension